docs/mkdocs.yml

```diff
@@ -81,12 +81,27 @@ nav:
       - Summarization: tutorials/summarization.md
       - Additional Resources: tutorials/other_tutorial.md
       - Examples: tutorials/examples.md
-  - Community:
+  - '' : '' 
+  - '' : ''
+  - '' : ''
+  - '' : ''
+  - '' : ''
+  - '' : ''
+  - '' : ''
+  - '' : ''
+  - '' : ''
+  - '' : ''
```

Comment:
Is this intentional? What is this doing?

Reply:
yeah @okhat  told me to do that way to create space  between tutorial and community ,FAQ